types/find-test-names/index.d.ts

```diff
@@ -32,6 +33,7 @@ export interface TestInfo {
 }
 export interface Suite {
     name: string;
+    requiredTags: string[] | undefined;
```

Comment:
Shouldn't this be optional? Same in Test.
https://github.com/bahmutov/find-test-names#requiredtags

Reply:
yes, the outcome would be same though, let me amend

Reply:
amended

Reply:
> yes, the outcome would be same though, let me amend

Not exactly, this way you don't need to specify it  in options at all ;)